### DISCUSSION TITLE
Using content nextjs in another application

### DISCUSSION THREAD

**Discussion by Devisione** [vercel/next.js#39213] [Ideas]
Hi all!

### **What a problem?**
 I have a task to integrate content from one next.js application into another next.js application.

With this integration, state hydration stops working due to the fact that page 2 has id __next and it tries to hydrate the first one. although everyone has to hydrate their block.

### **Solution**

Let's add to the configuration an indication in which element to render content?

**Comment by devtombiz** [comment]
Hi @Devisione were you able to make this work ? I am having a similar issue and can't find a solution so far.
